susemanager-utils/susemanager-sls/salt/packages/redhatproductinfo.sls

```diff
@@ -11,6 +11,10 @@ oraclerelease:
   cmd.run:
     - name: cat /etc/oracle-release
     - onlyif: test -f /etc/oracle-release
+amazonrelease:
+  cmd.run:
+    - name: cat /etc/system-release
+    - onlyif: test -f /etc/system-release
```

Comment:
When you change this, you should check the java part of the product detection. I think it requires also changes.

Reply:
Well, I did this in preparation for the Java product detection (that someone else will handle later). As far as I could see, it doesn't break anything if it's already done, right?

Reply:
Right. If product detection is not yer wanted we can leave it in.